src/client.cpp

```diff
@@ -431,139 +472,97 @@ void CClient::SetDoAutoSockBufSize ( const bool bValue )
     CreateServerJitterBufferMessage();
 }
 
-// In order not to flood the server with gain or pan change messages, particularly when using
+// In order not to flood the server with gain change messages, particularly when using
```

Comment:
Seems like you revert some recent changes like the pan timer.

Reply:
Eesh, that's possible, I tried to avoid that but with the length of time this took me, I may have missed a few commits.

Reply:
I see a few other recent changes in client.cpp, such as https://github.com/jamulussoftware/jamulus/commit/49eb6080303190b8d2c1fd73bfb38f3a085f34dc. I'm not sure what this client<->server channel mapping is about, or what server channels are, or what the requirement for this was.
For things like this, it might be best to cherry-pick the relevant commits, as the number of files affected will be minimal.